src/backend/auth/adapter.ts

```diff
@@ -35,6 +35,15 @@ export const createPersistedAuthAdapter = (
   db: KyselyDatabaseInstance
 ): Adapter => ({
   async createUser(user) {
+    const numOfUsers = await db
+      .selectFrom("user")
+      .select((qb) => qb.fn.count("user_id").as("count"))
+      .executeTakeFirstOrThrow();
+
+    // First created user is an admin
+    // todo: this should be pulled from an invite or something else instead of defaulting to a para - currently devs signing in are being assigned as paras
```

Comment:
I think we will want an admin panel where admins can designate others as admins.

We will likely not allow upgrading a user to admin from emails alone - we want email invites to be shareable.

Reply:
> I think we will want an admin panel where admins can designate others as admins.

Yep, that can be in a future PR.

> We will likely not allow upgrading a user to admin from emails alone - we want email invites to be shareable.

Not quite sure what you mean by this?